### DISCUSSION TITLE
I love react hook form.

### DISCUSSION THREAD

**Discussion by vandercloak** [react-hook-form/react-hook-form#11451] [General]
I just wanted to take a second and say that I love this library. I have used it on all the startups I have been involved with. 

I frequently think to myself, "I wonder if react hook form does X". Then without leaving vscode, I just try X, and you guys have built something so intuitive and familiar that I can just **_guess my way to getting to the right api syntax._** 

Example: 
I wanted the setFocus to highlight the text on an input... so I think, a lot of the api for react hook form has an options field at the tail end. Lets try that. Sure enough, it does. And whats in it? `shouldSelect`. 

Its a very impressive feat to build something this familiar + intuitive. Well done to everyone who had the vision for this and to those that implemented it 🙌 ❤️

**Comment by Moshyfawn** [comment]
Hi Nick!

Thanks for your kind words 💜 We may have limited time to respond in a timely manner, but your sentiments are greatly appreciated and have personally made my day of putting out fires at work brighter!

Of course, user feedback on the Library API is also invaluable to us, so thank you for taking the time to share your thoughts.

**Comment by OneTuskedMario** [comment]
Honestly, same. This library is so f**ing good!!
